networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/reputation/ReputationManager.java

```diff
@@ -13,156 +13,45 @@
 
 package tech.pegasys.teku.networking.p2p.reputation;
 
-import com.google.common.base.MoreObjects;
-import java.util.EnumSet;
 import java.util.Optional;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import org.hyperledger.besu.plugin.services.MetricsSystem;
-import tech.pegasys.teku.infrastructure.collections.cache.Cache;
-import tech.pegasys.teku.infrastructure.collections.cache.LRUCache;
-import tech.pegasys.teku.infrastructure.metrics.TekuMetricCategory;
-import tech.pegasys.teku.infrastructure.time.TimeProvider;
-import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 import tech.pegasys.teku.networking.p2p.network.PeerAddress;
 import tech.pegasys.teku.networking.p2p.peer.DisconnectReason;
-import tech.pegasys.teku.networking.p2p.peer.NodeId;
 
-public class ReputationManager {
-  // This is not a big ban, we expect the peer could be usable very soon
-  static final UInt64 COOLDOWN_PERIOD = UInt64.valueOf(TimeUnit.MINUTES.toSeconds(2));
-  // It's a big ban, we expect that the peer is not useful until major changes, for example,
-  // software update
-  static final UInt64 BAN_PERIOD = UInt64.valueOf(TimeUnit.HOURS.toSeconds(12));
+public interface ReputationManager {
+  ReputationManager NOOP =
+      new ReputationManager() {
```

Comment:
I think returning true for the NOOP operations makes more sense aka allowing everything.